defaults/validate.go

```diff
@@ -94,6 +94,22 @@ func Validate(value interface{}) error {
 }
 
 func validate(validation, fieldName string, p, v reflect.Value) error {
+	// Multiple validations separated by a colon (as in "empty;file") behave like an OR clause
+	if strings.Contains(validation, ";") {
+		individualValidations := strings.Split(validation, ";")
```

Comment:
I am thinking that calling `Split` directly and checking if we get more than a single result would be enough?

Reply:
Mmmm we could, but I feel like
```
if individualValidations := strings.Split(validation, "|"); len(individualValidations) > 1 {
```
is a bit less readable than what we have now, and
```
individualValidations := strings.Split(validation, "|")
if len(individualValidations) > 1 {
```
 is basically the same, but it makes my brain slightly uncomfortable, it's like it's the other way around :joy: 
 
 If you feel strong about this, or if you were thinking about something else, please let me know!

Reply:
Nope, I just prefer to keep the scope level down when feasible. It's a preference really so not at all blocking.